<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="totalNum">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>29</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>猴子总数</string>
    </property>
   </widget>
   <widget class="QLabel" name="outNum">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>29</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>出列编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outField">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>29</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>29</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalField">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>29</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>79</y>
      <width>921</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outcomeBrowser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>921</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>猴子选大王</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
